### DISCUSSION TITLE
A healthy Tensorboard log for Tacotron.

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#18] [Show and tell]
Below is how a healthy training looks like with Tacotron. I put it here as a reference for people using TTS. 

This is Tacotron2 with LJspeech dataset and gradual training is enabled. 

![image](https://user-images.githubusercontent.com/1402048/110213400-28d75580-7ea0-11eb-8ffb-27e3bc0789ce.png)

![image](https://user-images.githubusercontent.com/1402048/110213409-2d037300-7ea0-11eb-96d7-7678559902cf.png)

![image](https://user-images.githubusercontent.com/1402048/110213415-3260bd80-7ea0-11eb-9269-c630b73f1a24.png)

**Comment by thorstenMueller** [comment]
Hi @erogol .
Thanks for sharing tensorboard (_best practice_) graphs 👍 .

Maybe it might be helpful for beginners to tell that gradual training switches learning parameters based on a reached step count. This impacts the graphs shown here.

In this case (_i guess_) gradual switched values on:
* 50k steps
* 130k steps
* 280k steps

**Comment by erogol** [comment]
ignore (move the thread up again)
